docs/distribution/launcher.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: developer-doc
+title: Enso Launcher
+category: distribution
+tags: [distribution, launcher]
+order: 4
+---
+
+# Enso Launcher
+The launcher is used to run Enso commands (like the REPL, language server etc.)
+and seamlessly manage Enso versions. This document describes it's features. Its
+command-line interface is described in the [CLI](./launcher-cli.md) document.
+
+<!-- MarkdownTOC levels="2,3" autolink="true" -->
+
+- [Launcher Distribution](#launcher-distribution)
+- [Project Management](#project-management)
+  - [Creating a Project](#creating-a-project)
+  - [Per-Project Enso Version](#per-project-enso-version)
+  - [Project Configuration](#project-configuration)
+- [Enso and Graal Version Management](#enso-and-graal-version-management)
+  - [GraalVM Override](#graalvm-override)
+  - [Downloading Enso Releases](#downloading-enso-releases)
+  - [Downloading GraalVM Releases](#downloading-graalvm-releases)
+- [Running Enso Components](#running-enso-components)
+- [Global User Configuration](#global-user-configuration)
+- [Updating the Launcher](#updating-the-launcher)
+  - [Minimal Required Launcher Version](#minimal-required-launcher-version)
+  - [Downloading Launcher Releases](#downloading-launcher-releases)
+
+<!-- /MarkdownTOC -->
+
+## Launcher Distribution
+The launcher is distributed as a native binary for each platform (Windows,
```

Comment:
You need to write your discoveries about static binaries and native image somewhere.